mpa/cls/trainer.py

```diff
@@ -172,7 +175,8 @@ def train_worker(gpu, dataset, cfg, distributed, validate, timestamp, meta):
                         num_gpus=len(cfg.gpu_ids),
                         dist=distributed,
                         round_up=True,
-                        seed=cfg.seed
+                        seed=cfg.seed,
+                        drop_last=train_data_cfg.drop_last if train_data_cfg.get('drop_last', False) else False
```

Comment:
I think it would be better to subtract the 'drop_last' variable so that it is declared separately.

Reply:
Do you mean like below:
`drop_last = train_data_cfg.drop_last if train_data_cfg.get('drop_last'. False) else False
...
build_dataloader(
...
drop_last=drop_last
)
`